supervision/keypoint/annotators.py

```diff
@@ -357,7 +368,7 @@ def annotate(
                 org=(box[0], box[3]),
                 fontFace=font,
                 fontScale=self.text_scale,
-                color=self.text_color.as_rgb(),
+                color=text_color.as_rgb(),
```

Comment:
this should be `as_bgr`. OpenCV has a blue-green-red ordering of colors. You can find that we call other opencv functions similarly.